salt/metalk8s/orchestrate/deploy_salt_minion_on_new_node.sls

```diff
@@ -0,0 +1,23 @@
+{%- set control_plane_ips = salt.saltutil.runner('mine.get', tgt='*', fun='control_plane_ips') %}
+{%- if 'bootstrap' in control_plane_ips.keys() and control_plane_ips['bootstrap'] %}
+{%- set control_plane_ip = control_plane_ips['bootstrap'][0] %}
```

Comment:
Wow "bootstrap" really hardcoded everywhere

Reply:
We can't even assume this is the correct host/minion name...

Reply:
I'll fix it as Christian has been doing it.